src/javascripts/components/hub/hub.js

```diff
@@ -12,10 +16,138 @@ const removeAirport = (e) => {
     .catch((err) => console.error('cannot remove airport', err));
 };
 
+const createAirport = (e) => {
+  e.preventDefault();
+  const radio = $('#radio').is(':checked');
+  const newAirport = {
+    airportName: $('#name').val(),
+    imgUrl: $('#image').val(),
+    city: $('#city').val(),
+    state: $('#state').val(),
+    country: $('#country').val(),
+    airportCode: $('#airportCode').val(),
+    numFlights: $('#numFlights').val(),
```

Comment:
all the things that start with 'num' should save a numbers - you will need to multiply each one by the number 1